### ISSUE TITLE
Convert Myplanet site to install profile to demo the project cookbook format

### ISSUE THREAD

**Issue by patcon** [oriuminc/vagrant-ariadne#28]
The [Myplanet site](http://www.myplanetdigital.com) (stripped of private credentials) will be converted to an install profile. This will be used to showcase how ariadne can work to help build install profiles.

Project install profile: http://github.com/myplanetdigital/myplanet
Aradne project cookbook: https://github.com/myplanetdigital/ariadne-myplanet

As an aside, we'll be using this to help all developers have access to improving the company site.

**Comment by mparker17** [comment]
Is this done?

**Comment by patcon** [comment]
Nope, but the site is stabilized, so I've been plucking away at stuff on weekends

ariadne-myplanet cookbook is no longer required

Was trying to get myplanetdigital/myplanet working with travis-ci:
https://github.com/myplanetdigital/myplanet/blob/develop/.travis.yml

Was having a bit of trouble getting things working correctly (mostly sendmail issues during site-install), so instead went the route of taking someone else's repo (sprice/classic) that had travis-ci working, and slowly morphing it into using our scripts:
https://github.com/myplanetdigital/classic

Might take the route of just copying all the myplanetdigital/myplanet repo files into that project and renaming it, as I never did figure out the issue that was causing travis not to work on the original myplanetdigital/myplanet repo

Anyhow, that was a bit of an all-over-the-place explanation, but just wanted to explain where it was -- experience with a recent project has made me really gung-ho on writing casperjs or behat tests for install profiles. Definitely ping me if you want to get together and troop through the actual install profile conversion on an awesomeday :)

**Comment by mparker17** [comment]
Sounds cool! I'd love to do that.
